apps/api/src/domain/models/playlist/commands/translate-playlist-name/translate-playlist-name.command.ts

```diff
@@ -0,0 +1,26 @@
+import { ICommandBase, LanguageCode } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, NonEmptyString } from '@coscrad/data-types';
+import { LanguageCodeEnum, PlayListCompositeId } from '../create-playlist.command';
+
+@Command({
+    type: 'TRANSLATE_PLAYLIST_NAME',
+    label: 'translate playlist name',
+    description: 'translating playlist names',
+})
+export class TranslatePlaylistName implements ICommandBase {
+    @NestedDataType(PlayListCompositeId, {
+        label: 'Composite Identifier',
+        description: 'system-wide unqiue identifier',
+    })
+    readonly aggregateCompositeIdentifier: PlayListCompositeId;
+
+    @LanguageCodeEnum
```

Comment:
Seems like we should be passing through the data types options so we can add a label and description specific to this property.

Reply:
done